### DISCUSSION TITLE
What's happening (April 2022)

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#203] [General]
Monthly update of information for the Catalyst browser

### Version format change
Instead of every release being for example 1.3, 1.4, 1.5. It will increment by one every release, similar to chrome.
### LTS releases
Every 5 releases, will be an LTS release (Long Term Support). So if you prefer and older release but want it patched and secure, get the latest LTS release.
### Catalyst Lite (WIP)
A fast and lightweight version of Catalyst (loads 5x faster), [https://github.com/JaydenDev/catalyst-lite](@ GitHub)

Thats it for April 2022, questions? ask here.

**Comment by michaeleldar** [comment]
I know this is old, but if you still do LTS releases, when was the last one?
